docs/baseline.md

```diff
@@ -175,6 +178,6 @@ Here are some areas for future consideration and not currently in-scope for Base
 * Upcoming (e.g., not yet shipped in all browsers)
 * Progressive enhancement safe (i.e., limited penalties for support failures)
 * Developer feedback requested
-* Buggy (e.g., supported but in ways that are surprising and semi-interoperable)
+* Buggy (e.g., supported but in ways that are surprising and semi-interoperable, not exposed to platform accessibility APIs (AAPIs) as specified)
```

Comment:
I think I'd make accessible a separate bullet point, since something could be fully interoperable for non-AT users and fully unsupported in AT (I hope we don't have features in that state, but I still think it makes sense to make "accessible" a top level item rather than a sub-case of generally buggy).

Reply:
Done. To be clear, I don't necessarily think that Baseline should have a dedicated state for that. I was more using the list to make explicit that exposure to platform accessibility APIs is one of the areas that should be taken into account once we have enough support data.